src/stc/scintilla/lexers/LexPowerPro.cxx

```diff
@@ -10,7 +10,7 @@
 //
 // Changes:
 // 	2008-10-25 - Initial release
-//	2008-10-26 - Changed how <name> is hilighted in  'function <name>' so that
+//	2008-10-26 - Changed how <name> is highlighted in  'function <name>' so that
```

Comment:
I have examined the [Scintilla's license](http://www.scintilla.org/License.txt), and it says that anyone can modify the code. So, I think there is nothing bad in this small fix.